keyboards/cipulot/chroma/hal_lld.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2023 Cipulot
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Load local stm32_registry after loading the ChibiOS one to patch ChibiOS bugs */
```

Comment:
This feels gross

Reply:
If the comment refers to the comment line in the code I can remove it w/o issues.

Reply:
No, the includes.  And I'm guessing there isn't an easy fix for this, currently.

Reply:
Was a bug raised with ChibiOS on their forum?

Reply:
Iirc it was, can’t pinpoint it rn tho. 
And actually for this PR I will swap the MCU from the G4 to F072. I between the PR creation and today I swapped it because of manufacturing availability of it. Will be flagging this as Draft so that I can swap it, otherwise I’d have to pass through develop for the change.